allennlp/modules/token_embedders/bag_of_word_counts_token_embedder.py

```diff
@@ -72,9 +89,12 @@ def from_params(cls, vocab: Vocabulary, params: Params) -> 'BagOfWordCountsToken
         we look for a ``vocab_namespace`` key in the parameter dictionary
         to know which vocabulary to use.
         """
+
         vocab_namespace = params.pop("vocab_namespace", "tokens")
-        vocab_size = vocab.get_vocab_size(vocab_namespace)
         projection_dim = params.pop_int("projection_dim", None)
+        ignore_oov = params.pop("ignore_oov", False)
```

Comment:
use `pop_bool` here, in case we want to pass that in as an environment variable or something